src/EventListener/WriteListener.php

```diff
@@ -25,10 +26,12 @@
 class WriteListener
```

Comment:
Also, I think we could mark this class as `final`? It was never designed to be extended in the first place...

(Strictly speaking, it'd be a BC break... But I highly doubt it'd bother anyone. I don't remember if we've done something like this before.)

Reply:
or `@internal` ?

Reply:
Let's add `@final` in the PHPDoc.

Reply:
See https://github.com/api-platform/core/pull/2046

Reply:
done

Reply:
Let's leave this to #2046 then. :smile: